broadcasting.md

```diff
@@ -208,44 +208,126 @@ window.Echo = new Echo({
+**1. Update token expiry (Default 3600, 1 hr)**
+- Update`ABLY_TOKEN_EXPIRY` in `.env` file. 
+- Update `ably` section under `config/broaadcasting.php` with `'token_expiry' => env('ABLY_TOKEN_EXPIRY', 3600)`
+
+**2. Modify channel capability**
+- Channel access can be changed as per [Channel Capabilities](https://ably.com/docs/core-features/authentication#capability-operations)
+```php
+  // file - routes/channels.php
+
+  // for private channel (array is returned as a truthy value)
+  Broadcast::channel('channel1', function ($user) {
+      return ['capability' => ["subscribe", "history"]];
+  });
+  
+  // for presence channel
+  Broadcast::channel('channel2', function ($user) {
+      return ['id' => $user->id, 'name' => $user->name, 'capability' => ["subscribe", "presence"]];
+  });
+```
+
+**3. Disable public channels**
+- Update `ABLY_DISABLE_PUBLIC_CHANNELS`, set as **true** in `.env` file. 
+- Update `ably` section under `config/broadcasting.php` with `'disable_public_channels' => env('ABLY_DISABLE_PUBLIC_CHANNELS', true)`
+
+<br/>
+
+> ### **Deprecated (Using ably-pusher adapter)** 
+>In this example, we will install the `pusher-js` package.
+>
+>You may wonder why we would install the `pusher-js` JavaScript library even though we are using Ably to broadcast our events. Thankfully, Ably includes a Pusher compatibility mode which lets us use the Pusher protocol when listening for events in our client-side application:
+>
+>```shell
+>npm install --save-dev laravel-echo pusher-js
+>`````
+>
+>**Before continuing, you should enable Pusher protocol support in your Ably application settings. You may enable this feature within the "Protocol Adapter Settings" portion of your Ably application's settings dashboard.**
+>
+>Once Echo is installed, you are ready to create a fresh Echo instance in your application's JavaScript. A great place to do this is at the bottom of the `resources/js/bootstrap.js` file that is included with the Laravel framework. By default, an example Echo configuration is already included in this file; however, the default configuration in the `bootstrap.js` file is intended for Pusher. You may copy the configuration below to transition your configuration to Ably:
+>
+>```js
+>import Echo from 'laravel-echo';
+>
+>window.Pusher = require('pusher-js');
+>
+>window.Echo = new Echo({
+>    broadcaster: 'pusher',
+>    key: process.env.MIX_ABLY_PUBLIC_KEY,
+>    wsHost: 'realtime-pusher.ably.io',
+>    wsPort: 443,
+>    disableStats: true,
+>    encrypted: true,
+>});
+>```
+>
+>Note that our Ably Echo configuration references a `MIX_ABLY_PUBLIC_KEY` environment variable. This variable's value should be your Ably public key. Your public key is the portion of your Ably key that occurs before the `:` character.
+>
+>Update php-Laravel config. - Set `ABLY_PUSHER_ADAPTER` as **true** in `.env` file. Also, make sure `ably` section under `config/broadcasting.php` is updated with `'pusher_adapter' => env('ABLY_PUSHER_ADAPTER', true)`
```

Comment:
Second part is not required, and isn't common practice in Laravel, as it's the default value and should be left at `false`. The purpose of the `.env` file is to allow users to use custom values instead of the default specified in `config/broadcasting.php` or another config file.

Reply:
Updated!